tests/network/conftest.py

```
@@ -68,24 +66,6 @@ def dual_stack_cluster(ipv4_supported_cluster, ipv6_supported_cluster):
     return ipv4_supported_cluster and ipv6_supported_cluster
 
 
-@pytest.fixture()
-def skip_if_not_ipv4_supported_cluster_from_mtx(
-    request,
-    ipv4_supported_cluster,
-):
-    if ip_version_data_from_matrix(request=request) == IPV4_STR and not ipv4_supported_cluster:
-        pytest.skip("IPv4 is not supported in this cluster")
```

Can we keep this logic and instead of skipping, let's assert, so the test will fail and not continue.